backend/server/routers/model.py

```diff
@@ -28,19 +28,28 @@ class CourseDetails(BaseModel):
     study_level: str
     school: Optional[str]
     campus: str
-    equivalents: dict
+    equivalents: dict[str, str]
```

Comment:
should be `dict[str, Literal[1]]`

Reply:
I think not - it is the name and code of the course
this is checked at runtime by pydantic soooo... should be ok if the tests pass lol